docs/reviewing.md

```diff
@@ -0,0 +1,36 @@
+Two output checkers independently review each file and record their decisions using the buttons within Airlock.
+Whilst viewing a file in an Airlock release request, they have the option of: 
+
+* **Approve** — output meets disclosure requirements and is safe to be released
+* **Reject** — output is not currently acceptable for release. 
+
+!!!warning "Retrictions for authors"
+    Airlock does not permit users to approve or reject files that are part of release requests they created themselves.
+
+
+## Viewing release requests on Airlock
+
+1. Output checkers navigate to the Requests list to view all open requests
+    awaiting review. Note that if an output checker is the author of a request, they
+    will not see it in the outstanding requests list. 
+1. Output checkers identify the request by workspace name and requester username.
```

Comment:
Will there only ever be one active request per workspace? Or is it one active request per user in a given workspace? Maybe worth mentioning here

Reply:
There will only every be one active request _per user_ per workspace. It's unlikely, but possible, for different users to have a release for the same workspace.  I'll add a clarification here.